data/World/Jabu Jabus Belly MQ.json

```diff
@@ -44,23 +45,24 @@
                 at('Jabu Jabus Belly Main', can_use(Hover_Boots) and can_use(Hookshot))"
         },
         "exits": {
-            "Jabu Jabus Belly Main": "True",
-            "Jabu Jabus Belly Boss Area": "Sticks or (can_use(Dins_Fire) and Kokiri_Sword)"
+            "Jabu Jabus Belly Main": "True"
+        },
+        "events": {
+            "Jabu Jabus Belly Floor Lowered": "Sticks or (can_use(Dins_Fire) and Kokiri_Sword)"
         }
     },
     {
-        "region_name": "Jabu Jabus Belly Boss Area",
+        "region_name": "Jabu Jabus Belly Before Boss",
```

Comment:
Just FYI, this region is referring to anything after big octo. The cow is in the room after fighting big octo.

Reply:
Probably doesn't change the naming at all, but I thought I'd let you know.